integrationtest/src/main/scala/io/gearpump/integrationtest/minicluster/RestClient.scala

```diff
@@ -44,6 +47,20 @@ class RestClient(host: String, port: Int) {
     callFromRoot("version")
   }
 
+  def listMasters(): MasterSummary = try {
```

Comment:
Masters should be a List with one element. What the method returns is `getMasterLeader`